front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -213,7 +202,10 @@ export function VisualizationActionIframe({
         </div>
       )}
       <div
-        className="transition-height relative min-h-96 w-full overflow-hidden duration-500 ease-in-out"
+        className={classNames(
+          "transition-height relative w-full overflow-hidden duration-500 ease-in-out",
+          codeFullyGenerated ? "min-h-96" : ""
```

Comment:
IIRC changing style like this won't rely on the CSS height animation

Reply:
I didn't notice a change but maybe you are right, I'm going to check. The main thing is I wanted to avoid the empty white space below the code block at the beginning of the generation

Reply:
Yep def makes sense to remove it.

Reply:
I think the animation still works fine, because the style is applied before the iframe gets rendered, so when it needs to grow / shrink to switch to iframe content, the min-h-96 is already here

Reply:
🙌